buildspec-py3.yml

```diff
@@ -152,6 +153,7 @@ phases:
         --build-arg VERSION="${GIT_TAG}"
         -t "${DOCKER_IMG_TAG}" -t "${DOCKER_IMG_TAG_LATEST}" .
       - echo Build completed on $(date)
+      - pipenv install -e .
```

Comment:
This should not be needed here.

Reply:
it is absolutely needed here, as the tests use the chsdi package. I could put it right before the tests, though

Reply:
See makefile comment